configure.ps1

```diff
@@ -103,6 +103,22 @@ If ($Test) {
     exit 0
 }
 
+# Use the MSVC linker to determine if the respective SpiderMonkey library
+# is available on the linker path.  This heuristic is taken from
+# src/couch/rebar.config.script, please keep them in sync.
+If ($SpiderMonkeyVersion -eq "1.8.5") {
+    $SpiderMonkeyLib = "mozjs185-1.0.lib"
+}
+else {
+    $SpiderMonkeyLib = "mozjs-$SpiderMonkeyVersion.lib"
+}
+
+&link $SpiderMonkeyLib /SUBSYSTEM:CONSOLE /NOENTRY /DLL /OUT:NUL *> $null
```

Comment:
Is link.exe always available or only if you invoke the VS dev cmd/powershell before?

Reply:
Yes, `c:\relax\couchdb-glazier\bin\shell.ps1` needs to be invoked before, otherwise `link` will correspond to GNU `ln(1)`.  But this needs to be done anyway, because e.g. `escript` will not work either.

Reply:
I ended up using `link` because that is [what `rebar` would also do on Windows](https://github.com/rebar/rebar/blob/b6d309417c502ca243f810e5313bea36951ef038/src/rebar_port_compiler.erl#L707).

Reply:
You are absolutely right of course. I do the invocation of shell.ps1 automatically if I start Powershell, so I forgot about it...